src/pages/not-found/not-found.module.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../styles/variables.module.scss';
+@import '../../styles/fonts.module.scss';
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 48px;
+  height: 100%;
+  width: 100%;
+  margin-top: -88px;
```

Comment:
This is an offset to overwrite the gap caused by the empty Layout title

Reply:
Maybe it's worth to write this comment also to the file itself?

Reply:
Removed the offset entirely after implementing Markus' feedback with the Layout `title`